src/model/composer/modifyselection.js

```diff
@@ -6,21 +6,44 @@
 import Position from '../position.js';
 import TreeWalker from '../treewalker.js';
 import Range from '../range.js';
+import { isInsideSurrogatePair, isInsideCombinedSymbol } from '../../../utils/unicode.js';
 
+/* jshint -W100 */
```

Comment:
We needed it before, because there was an incorrect sign in the comment (half of surrogate pair, rendered) and linter was throwing an error because it won't be properly displayed in HTML :).

Then the comment changed and we do not need it anymore.